Mathlib/Data/Finset/Card.lean

```diff
@@ -120,6 +120,26 @@ theorem card_insert_le (a : α) (s : Finset α) : card (insert a s) ≤ s.card +
   · rw [card_insert_of_not_mem h]
 #align finset.card_insert_le Finset.card_insert_le
 
+section
+
+variable {a b c d e f : α}
+
+theorem card_le_two : card {a, b} ≤ 2 := card_insert_le _ _
+
+theorem card_le_three : card {a, b, c} ≤ 3 :=
+  (card_insert_le _ _).trans (Nat.succ_le_succ card_le_two)
+
+theorem card_le_four : card {a, b, c, d} ≤ 4 :=
+  (card_insert_le _ _).trans (Nat.succ_le_succ card_le_three)
+
+theorem card_le_five : card {a, b, c, d, e} ≤ 5 :=
+  (card_insert_le _ _).trans (Nat.succ_le_succ card_le_four)
+
+theorem card_le_six : card {a, b, c, d, e, f} ≤ 6 :=
+  (card_insert_le _ _).trans (Nat.succ_le_succ card_le_five)
```

Comment:
This feels really clumsy; I wonder if in the long run we can do something like #5863 to avoid having uncountably many gaps in our API in places like this; so have a `Finset.card_le%` elaborator that generates these lemmas on the fly by reflection.

Reply:
For reference, we have [Fin.sum_univ_eight](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/BigOperators/Fin.html#Fin.sum_univ_eight), [Finset.card_eq_three](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Finset/Card.html#Finset.card_eq_three), [Finset.two_lt_card](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Finset/Card.html#Finset.two_lt_card), [Set.two_lt_ncard](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Set/Card.html#Set.two_lt_ncard) etc.

Reply:
I didn't know we went as high as eight!

Reply:
Those were added in [this commit](https://github.com/leanprover-community/mathlib/pull/15684/commits/39bf3e27455a777312de7dd0d8c20dab5e24ba89), and apparently you reviewed that PR after the commit ...

Reply:
I think having `Finset.card_le%` at a later stage is perfect. But I'm not the right person to write it.